client/components/login/login.js

```diff
@@ -62,10 +62,21 @@ export default class Login extends React.Component {
 
     // social login / SAML login
     if (username && token) {
+      const loginMethod = getParameterByName("login_method");
+      if (loginMethod) {
+        // we have to use localStorage because the page may be
+        // reloaded and the information may be lost
+        localStorage.setItem(`${orgSlug}_logout_method`, loginMethod);
```

Comment:
sessionStorage will work also.

Reply:
> sessionStorage will work also if it's only for retrieving information when the page is reloaded.

The user may close the window (especially if it's the minibrowser which automatically pops up) and load it again from another browser.

Reply:
PS: I will update the comment to reflect this.